### ISSUE TITLE
Train yolov12 on the Anti-UAV dataset

### ISSUE THREAD

**Issue by clodiamery** [ultralytics/ultralytics#22357]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

I train the yolov12 on the Anti-UAV dataset and I got on mAp50 = 83% accuracy. According to some papers that train yolov11 on the same dataset and got on mAp50 = 84.2% accuracy, is that correct? is the yolov12 better than yolov11?

**Comment by UltralyticsAssistant** [comment]
👋 Hello @clodiamery, thank you for your interest in Ultralytics 🚀! This is an automated response — an Ultralytics engineer will also assist you here shortly. In the meantime, we recommend a visit to the [Ultralytics Docs](https://docs.ultralytics.com/) for new users where you can find many [Python usage examples](https://docs.ultralytics.com/usage/python/) and [CLI usage examples](https://docs.ultralytics.com/usage/cli/); many common questions are already answered there.

If this is a custom training ❓ Question, please share as much detail as possible to help us understand your results and provide guidance:
-Exact model variant used (e.g., YOLO12n/s/m/l/x) and any config changes
-Dataset details (splits, class counts) and sample images
-Training settings (epochs, imgsz, batch size, augmentations, optimizer, seed)
-Validation protocol (val/test split, confidence/IoU thresholds) and full training logs
-Command(s) or code used to train/validate and the resulting metrics
Also, please verify you’re following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/) ✅

Join the Ultralytics community where it suits you best. For real-time chat, head to the [Ultralytics Discord](https://discord.com/invite/ultralytics) 🎧. Prefer in-depth discussions? Check out the [Ultralytics Discourse forum](https://community.ultralytics.com/). Or dive into threads on our [Ultralytics Subreddit](https://www.reddit.com/r/Ultralytics/) to share knowledge with the community.

## Upgrade

Upgrade to the latest `ultralytics` package including all [project requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [Python>=3.8 environment](https://www.python.org/) with [PyTorch>=1.8](https://pytorch.org/get-started/locally/) to verify your question isn’t already addressed by recent updates:

```bash
pip install -U ultralytics
```

## Environments

YOLO may be run in any of the following up-to-date verified environments (with all dependencies including [CUDA](https://developer.nvidia.com/cuda-zone)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See the [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See the [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See the [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI checks](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

**Comment by glenn-jocher** [comment]
Thanks for sharing—an ~1% mAP50 gap (83 vs 84.2) is within normal run-to-run variance unless splits, `imgsz`, epochs, augmentations, and evaluation are identical; YOLO12 isn’t universally better, and we still recommend [YOLO11 models](https://docs.ultralytics.com/models/yolo11/) as the default for stability, memory use, and CPU speed, while [YOLO12](https://docs.ultralytics.com/models/yolo12/) is more experimental. For an apples-to-apples check, please validate both checkpoints on the same split and report both mAP50 and mAP50-95 as shown by [Val mode](https://docs.ultralytics.com/modes/val/): 
```bash
yolo mode=val model=yolo11m.pt data=anti-uav.yaml imgsz=640
yolo mode=val model=path/to/your_yolo12_best.pt data=anti-uav.yaml imgsz=640
```
